### PR TITLE
feat: empty state container for targets page

### LINKED ISSUE #669: Targets page displays an empty page
The targets page displays an empty view on demo.parca.dev. This is because we're not scraping any targets but are actually receiving profiles being sent from parca-agent.

It'll be nice to show an empty state placeholder explaining this.

### PR DESCRIPTION
**Pull request by monicawoj**
resolves #669 

Creates a reusable EmptyState container component in the shared folder, which can be used throughout the application as a placeholder when data is not available

### CODE REVIEW

File: ui/packages/app/web/tailwind.config.js [hunk 1]
```diff
@@ -10,6 +10,9 @@ module.exports = {
       maxWidth: {
         '1/2': '50%',
       },
+      minWidth: theme => ({
+        ...theme('spacing'),
```
Comment (manojVivek) [inline ui/packages/app/web/tailwind.config.js:14]: Just curious, what does this do?

**Review by manojVivek — APPROVED** [review]
Lgtm!

File: ui/packages/app/web/tailwind.config.js (hunk 1 above)
Comment (monicawoj) [reply ui/packages/app/web/tailwind.config.js:14]: @manojVivek this extends the tailwind css potential values available for setting 'minimum width' by making them the same as the options available for setting 'width'. Min-width had very few available options. For example, 'min-w-64' was not available, but 'w-64' (which equates to 16rem) was. So basically now, all the options available for width are also available for min width.

**Comment by metalmatze** [comment]
Amazing! 😊 🎉

File: ui/packages/app/web/tailwind.config.js (hunk 1 above)
Comment (manojVivek) [reply ui/packages/app/web/tailwind.config.js:14]: Got it, thanks for the explanation, Monica. 
I wonder why wouldn't they do it by default. 🤔

**Pull request merged**